### ISSUE TITLE
[feature request] add camera Zoom

### ISSUE THREAD

**Issue by primarto** [Fr4gorSoftware/SecScanQR#153]
it would be good if it can zoom in with the camera, it can help in some situations. 
I saw this in another app on fdroid, but I like more to use your app.

**Comment by StevanWhite** [comment]
I agree.  Here's the reason: many cheaper phones do not have "auto focus".  To make the image big enough to fit the scannar, the phone has to be brought too close to the code to focus.

With a simple zoom, smaller codes could be recognized by such phones.

**Comment by liamdennehy** [comment]
Long overdue 😉
